templates/viewer.php

```diff
@@ -95,6 +95,7 @@
       </div>  <!-- sidebarContainer -->
 
       <div id="mainContainer">
+        <div id="save-progress"><?php p($l->t('Saving document.')); ?></div>
```

Comment:
Instead of showing the message at the top of the PDF viewer while the file is being saved it could be better to change the button icon to a loading spinner. The rules for the `icon-loading-small` CSS class are defined in the [Nextcloud server styles](https://github.com/nextcloud/server/blob/cc6037e438e5302320d9b864021fc3d5e42a3180/core/css/icons.scss#L45-L116), but as the viewer is shown in an iframe those styles are not available. While it might be possible to import the stylesheets inside the iframe with some JavaScript the easiest solution would be to copy the relevant CSS rules to [_css/viewer.css_](https://github.com/nextcloud/files_pdfviewer/blob/8830639acef86663c739d69aa5c58cdb54d85141/css/viewer.css), which is the CSS file used by the PDF viewer inside the iframe. The icons for the toolbar buttons are [a mask](https://github.com/mozilla/pdf.js/blob/v3.9.179/web/viewer.css#L925) applied on [a background colour](https://github.com/mozilla/pdf.js/blob/v3.9.179/web/viewer.css#L838), so unsetting the background colour when the loading icon is shown would hide the normal icon.